### ISSUE TITLE
[GlobalStep] Wrong Icon displayed on “View Site” button

### ISSUE THREAD

**Issue by wptester9845** [wordpress-mobile/WordPress-Android#11320]
### Description
While the “View Admin” button displays an icon indicating the page is opened on an external Web Browser, the “View Site” button displays the same icon despite being opened inside the WordPress application.

### Reproduction Rate
4/4 100%

### Expected behavior
The aforementioned icon should not be present.

### Actual behavior
The icon indicating the page is opened on an external Web Browser is displayed.

### Steps to reproduce the behavior
1. Install WordPress 14.2
2. Log in to an account.
3. Select a Site.
##### Tested on the following
Samsung Galaxy J3 (8.0.0)

#### Please see the attached video for more information
[AndroidExternal.zip](https://github.com/wordpress-mobile/WordPress-Android/files/4205692/AndroidExternal.zip)

### Submitted by:
Luis Pimenta

**Comment by designsimply** [comment]
<img src="https://user-images.githubusercontent.com/1119271/74785589-de207680-5267-11ea-9e76-e1a041a961c3.png" width="200" alt="Screenshot_20200218-155827"> <img src="https://user-images.githubusercontent.com/1119271/74785530-c3e69880-5267-11ea-9051-ef92527dac51.png" width="200" alt="Screenshot_20200218-155841"> <img src="https://user-images.githubusercontent.com/1119271/74785522-c2b56b80-5267-11ea-9d2f-5a458f3ba882.png" width="200" alt="Screenshot_20200218-155848">
<sup>Tested with WPAndroid 14.2-rc-2 on Pixel 3 Android 10.</sup>

I tested and found that "View Site" opens in a WebView which I think we consider something between the app and an external browser and "View Admin" prompts the user to open the link in an external browser such as Chrome.

The icon may be okay for both. @osullivanchris can you provide a decision for us about whether or not the icon needs to change for the "View Site" WebView link in My Sites since it technically opens in a preview and not in an external browser?

**Comment by osullivanchris** [comment]
@designsimply I agree its somewhat in between. Seems good to indicate that you are leaving the main app experience.

On iPhone we do per the request here and show the icon for 'View admin' but not for 'View site'. For 'View Site' we use the standard chevron (which is a standard list item thing on the rows on iOS, we don't use it on Android)

As the title of the section says "External" I think it already implies that you will be leaving the main experience. But having that icon on both implies they have the same level of 'leaving' for want of a better term.

Therefore I think I agree with the proposal. If we remove this icon for 'View Site', its still in the 'external' section. But 'View Admin' would get that extra affordance that you are leaving the app altogether

**Comment by megsfulton** [comment]
+1 for the proposal to remove the icon for view site. Removing the "Needs design" label
